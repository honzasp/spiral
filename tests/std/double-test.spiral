(program
  (import std)
  (println 0.5)
  (println (+ 1.5 0.25))
  (println (- 3.5 2.25))
  (println (* 0.5 5.0))
  (println (/ 5.0 2.5))

  (println (+ 1 1.5))
  (println (+ 2.5 3)))
