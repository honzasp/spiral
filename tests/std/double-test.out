0.5
1.75
1.25
2.5
2
2.5
5.5
